src/fidesops/service/connectors/saas_query_config.py

```diff
@@ -160,7 +162,7 @@ def generate_update_stmt(
         This masked row is then added as the body to a dynamically generated SaaS request.
         """
 
-        current_request: SaaSRequest = self.get_request_by_action("update")
+        current_request: SaaSRequest = self.masking_request
```

Comment:
This solution now can feed multiple types of `SaaSRequests` into `generate_update_stmt`.  Delete and GDPR deletes still share a lot of the same code as an update request - for example, Delete requests can still have request bodies, may need identity data as param values or to use connector params, etc.  

As we get more use cases I think we can add more conditional logic to `generate_update_stmt`, but I think just sharing `generate_update_stmt` among the different types of SaaSRequests is a good first step.